plv8.cc

```diff
@@ -285,6 +302,18 @@ _PG_init(void)
 							NULL);
 #endif
 
+	DefineCustomIntVariable("plv8.memory_limit",
+							gettext_noop("Per-isolate memory limit in MBytes"),
+							gettext_noop("The default value is 256 MB"),
+							(int*)&plv8_memory_limit,
```

Comment:
256MB might be a little low, especially when dealing with ETL.  I'd feel more comfortable att around 1GB.  while it's not as big of a deal with the direct JSONB conversions, there are still some inefficiencies that are possible.

Reply:
it's only a default can be set to anything in the config file
testing oom with 1GB will take much longer